Readme.md

```diff
@@ -152,6 +152,16 @@ in double quotes, e.g. `qtuicargs { "foo", "bar" }` will appear as `"foo" "bar"`
 An optional list of arguments that will be sent to the Qt rcc tool. Each argument will be encased
 in double quotes, e.g. `qtrccargs { "foo", "bar" }` will appear as `"foo" "bar"` in the command line.
 
+##### qttsargs { "arg", "arg", ... }
+
+An optional list of arguments that will be sent to the Qt lrelease tool. Each argument will be encased
+in double quotes, e.g. `qtuicargs { "foo", "bar" }` will appear as `"foo" "bar"` in the command line.
```

Comment:
copy-paste: this should be `qtlreleaseargs { "foo", "bar" }`.

Reply:
Renamed.